### ISSUE TITLE
memory file panel and  international

### ISSUE THREAD

**Issue by mbaillif** [reds-heig/logisim-evolution#346]
I notice that the memory file panel is not internationalized. 
Is it possible to modify it to be able to present it in other languages ?

**Comment by BFH-ktt1** [comment]
Yes of course, and I agree that there are more parts, where only Englisch is available (for example FPGA-commander, some preferences, etc.). I'll put it as todo's, thanks for reporting them.

**Comment by hewzhew** [comment]
This request appears fully addressed by merged PR #2638.

That PR localized the memory image file chooser/panel labels, tabs, status text, save-preview tab, and file-filter labels. It deliberately kept the internal format descriptors canonical, so localization does not change the headers written to or parsed from memory files, and added focused regression coverage for that separation.

The previously noted local limitation was only the missing external `xxd` executable for binary-format tests. With the repository's Windows test prerequisite available, the complete 90-case `HexFileTest` suite now passes, including all 16 binary-format parameterizations.

I therefore think #346 can be closed as completed by #2638.
